common/models/role.js

```diff
@@ -179,8 +179,8 @@ module.exports = function(Role) {
     }
     var modelClass = context.model;
     var modelId = context.modelId;
-    var userId = context.getUserId();
-    Role.isOwner(modelClass, modelId, userId, callback);
+    var user = context.getUser();
+    Role.isOwner(modelClass, modelId, user.id, user.principalType, callback);
   });
```

Comment:
see below, fetching the userId from accessContext is no longer enough, consequently I had to introduce a `getUser()` method that also returns the user's principalType, and change the signature of `Role.isOwner()` to include the principalType as an optional parameter